src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -451,11 +451,11 @@ class FastSync(
 
       case n: BranchNode =>
         val hashes = n.children.collect { case Some(Left(childHash)) => childHash }
-        blockchain.saveNode(ByteString(n.hash), n.toBytes, syncState.targetBlock.number, withSnapshotSave = false)
+        blockchain.saveNode(ByteString(n.hash), n.toBytes, syncState.targetBlock.number)
```

Comment:
same here with this `ByteString(n.hash), n.toBytes, syncState.targetBlock.number`